pool/web/handler/admin_experiments_mailing.ml

```diff
@@ -348,10 +348,12 @@ end = struct
 
   let search_info =
     (fun req ->
-      Guard.ValidationSet.Or
-        [ experiment_effects MailingCommand.Create.effects req
-        ; combined_effects MailingCommand.Update.effects req
-        ])
+      let open Guard.ValidationSet in
```

Comment:
The check, if the `id` is in the URL, is not necessary. `search_info` always uses the same url: `admin/experiments/<experiment-id>/mailings/search-info`

IMO it is also cleaner to create different middlewares for different requirements